week2/index.js

```diff
@@ -0,0 +1,213 @@
+const fs = require('fs');
+const readline = require('readline');
+let todos = [];//to be filled todo list
+
+//check if there is a parameter
+if(process.argv[2] !== undefined){
+    start();
+} else {
+    console.log('Please specify an option to use, here is the manual\n');
+    manualReader('./help.txt');
+}
+
+
+function start(){
+    switch(process.argv[2].trim().toLowerCase()){
+        case 'help':
+            manualReader('./help.txt');
```

Comment:
I like that you break out into another function call here, instead of putting things inline. That makes it easy to see what the different options are, and where to go look for the corresponding code.